frontend/src/components/GPopper.vue

```diff
@@ -96,6 +96,10 @@ export default {
     },
     disabled: {
       type: Boolean
+    },
+    boundariesSelector: {
+      type: String,
+      default: '.v-content__wrap'
```

Comment:
BTW: Do we have a test that checks, if the v-content__wrap element is available (i.e. do we have a way to ensure that vuetify does not change the name), if not we should take the opportunity and add it.

Reply:
add test vor v-content and check for existence of v-content__wrap

Reply:
✅